kolibri_explore_plugin/assets/src/customApps.js

```diff
@@ -28,7 +28,8 @@ export const CustomChannelApps = {
   '59bb2e5a3d2e5e3b85b87d9ab4daa2f3': 'reading',
   '3fcffebc58d15175b948b140434ef6e6': 'sports',
   '2f95235c3709511fa12d007f31ed6a7b': 'steam',
-  e5b26962b0ee5d8487152d0987d32e44: '49ers-edu-digital-playbook',
+  bf0260ed911f44cda27a263db93a8512: '49ers-edu-digital-playbook',
+  '1e755450dede5c40af1487e07984c260': 'techbridge-girls',
```

Comment:
For consistency, can you name the custom app `techbridge-girls-home` just like it's already used in other constants below? This means you will have to rename the new subfolder inside the `channel-overrides/` folder.

Reply:
Sure, done. Thanks for catching